src/imagetilesource.js

```diff
@@ -114,8 +114,9 @@
             }
 
             $.addEvent(image, 'load', function () {
-                _this.width = image.naturalWidth;
-                _this.height = image.naturalHeight;
+                /* IE8 fix since it has no naturalWidth and naturalHeight */
+                _this.width = Object.prototype.hasOwnProperty.call(image, 'naturalWidth') ? image.naturalWidth : image.width;
```

Comment:
I'm not sure why, but `image.hasOwnProperty('naturalWidth')` doesn't work fine in IE8 when i debug i find [this question on OS](http://stackoverflow.com/questions/8157700/object-has-no-hasownproperty-method-i-e-its-undefined-ie8).
That's why i insist on this way of checking. But i agree, that's it's not so redable as we want it to be.